src/public/js/script.js

```diff
@@ -145,21 +154,20 @@ function hideShowDiv(val) {
   }
 }
 const formId = document.getElementById('my_form')
-document.getElementById('submit').addEventListener('click', function (event) {
+document.getElementById('submit').addEventListener('click', async function(event) {
   event.preventDefault()
   if (!form.checkValidity()) {
     event.preventDefault()
     event.stopPropagation()
     form.classList.add('was-validated');
   } else {
-    postData()
+    uniqueUuid = generateUniqueString(10)
+    await postData()
+    await addDesiredLanes()
```

Comment:
Why not with the same request? What if second one will fail?

Reply:
done